packages/circuit-ui/components/ActionMenu/ActionMenu.mdx

```diff
@@ -0,0 +1,28 @@
+import { Meta, Status, Props, Story } from '../../../../.storybook/components';
+import * as Stories from './ActionMenu.stories';
+
+<Meta of={Stories} />
+
+# ActionMenu
+
+An ActionMenu displays a list of subsequent actions, when interacting with an actionable component.
+
+<Story of={Stories.Base} />
+<Props />
+
+- The reference element, which triggers ActionMenu, can be primary, secondary, tertiary buttons, an overflow icon, or components with embedded buttons such as the [ImageInput](Forms/ImageInput) component.
+- Each ActionMenu action item is represented by an appropriate HTML element (e.g., a button element or an anchor element).
```

Comment:
Rendering the appropriate semantic element is handled internally. What's relevant to developers is that they should include `href` and/or `onClick` props.